apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -73,11 +73,12 @@ const requestResetPassword = createAsyncThunk<
 	EmailDto,
 	AsyncThunkConfig
 >(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
-	const { authApi, notification } = extra;
+	const { authApi, notification, storage } = extra;
 
 	const isSuccessful = await authApi.requestResetPassword(emailPayload);
 
 	if (isSuccessful) {
+		await storage.drop(StorageKey.TOKEN);
```

Comment:
If we are already in this action, I believe the user does not have token. If the user opens the app with an expired token, then status 401 will be received and the token will be removed by handleErrorMiddleware

Reply:
What if user tries to reset password when the token is not expired? Or we shouldn't handle this case?

Reply:
If the user requests a link, then there's no token, there's no user logged in. Reset password screen can only be accessed when there's no user